### PR TITLE
GH-864 Adds CollapsibleList layout to 1.8.0

### PR DESCRIPTION
**Pull request by jbrundage**
Fixes GH-864

@GordonSmith @mlzummo Please review

https://rawgit.com/jbrundage/Visualization/GH-864-CollapsibleList/demos/dermatology.html?src/layout/CollapsibleList

Signed-off-by: Jay Brundage <jaman.brundage@lexisnexis.com>

### CODE REVIEW

File: src/layout/CollapsibleList.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                    widget.target(li);
+                } else {
+                    return widget._target;
+                }
+                return li;
+            })
+        ;
+        rows.exit().remove();
+        rows.each(function(widget){
+            widget.render();
```
Comment (jbrundage) [inline src/layout/CollapsibleList.js]: @GordonSmith The child widgets don't render when I remove this line. Any idea what I've done wrong?

Comment (GordonSmith) [inline src/layout/CollapsibleList.js]: I just tested by removing that code and its working fine?  Had you rebased on latest when you tested (as it needed https://github.com/hpcc-systems/Visualization/pull/843 to work)

**Comment by GordonSmith** [comment]
General comments:
Can we rename it to layout/Accordion
Can you test with a few more child widget types (like Table, Surface, MultiChart)?

**Comment by GordonSmith** [comment]
Needs rebase

**Comment by jbrundage** [comment]
@GordonSmith I've rebased and made the requested changes (Adding more child widget types and renaming to layout/Accordion), but there are conflicts?

**Comment by GordonSmith** [comment]
You need to get latest from upstream and then rebase again.

**Pull request closed without merging**